packages/flutter_tools/test/android/gradle_test.dart

```diff
@@ -2,13 +2,32 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import '../src/common.dart';
```

Comment:
Nit: sort the local imports like this after the package imports, in a new paragraph (separated by a newline)